components/brave_new_tab_ui/containers/newTab/settings/sponsoredImagesToggle.tsx

```diff
@@ -25,94 +25,229 @@ interface Props {
   rewardsEnabled: boolean
   adsEnabled: boolean
   canSupportAds: boolean
+  isExternalWalletConnected: boolean
 }
 
-const Container = styled.div`
-  background-color: var(--info-background);
-  border-radius: 8px;
-  padding: 8px;
-  align-items: center;
-  display: flex;
-  flex-direction: column;
-  gap: 10px;
-`
-
 const ToggleRow = styled.div`
   width: 100%;
   display: flex;
   align-items: center;
   justify-content: space-between;
-  padding: 12px 14px;
-  border-radius: 10px;
+  padding: 8px 0px;
   background-color: var(--background1);
 `
 
-const DescriptionRow = styled.div`
-  width: 100%;
-  display: grid;
-  grid-template-rows: fit-content(16px) 1fr;
-  grid-template-columns: 16px 1fr;
-  padding: 0 8px;
-  gap: 8px;
-  color: #339AF0;
+const ControlsContainer = styled.span`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+`
+
+const InfoIcon = styled.span`
+  position: relative;
+  margin: 0px 18px;
+
+  > .icon {
+    height: 17px;
+    width: auto;
+    vertical-align: middle;
+  }
+
+  .tooltip {
+    position: absolute;
+    bottom: 100%;
+    left: -200px;
```

Comment:
In RTL UI (which you can force and still keep english via `chrome://flags`), this is making the text get clipped
<img width="749" alt="image" src="https://user-images.githubusercontent.com/741836/215852966-4f03a5f6-de8b-4d74-880b-9f7c8dac6f00.png">

This could get fixed by the following (as I can't think of a better way right now!):
```css
[dir=rtl] & {
  right: -200px;
}
```

Unfortunately that puts the arrow in the wrong place, so that also needs repositioning similarly.
<img width="417" alt="image" src="https://user-images.githubusercontent.com/741836/215853845-16a97e91-7f4b-4fd3-a436-c6e89be7d405.png">

Reply:
@petemill `inset-inline-start` seems to work for both directions. WDYT?

Reply:
@petemill It also looks like toggles' bullet doesn't move correctly to the ON position with RTL set.